R/generate_wrappers.R

```diff
@@ -425,10 +371,9 @@ prepareCodilutionData <- function(df, df_layout) {
   colnames(df) <- paste0(colnames(df), "_2")
   df_2 <- cbind(df_layout, df)
   df_2 <- df_2[df_2$DrugName != df_2$DrugName_2, ]
-  rows <- df_2$Concentration_2 > 0
+  rows <- which(df_2$Concentration_2 > 0)
```

Comment:
safer solution?

Reply:
`DT` requires indexes for rows instead of boolean values

Reply:
Should we add the task to evaluate if there are other such cases left in our code?

Reply:
Yes, we can evaluate it

Reply:
Please add the task.